lib/game/game_widget.dart

```diff
@@ -82,18 +92,19 @@ class GameWidget<T extends Game> extends StatefulWidget {
     this.loadingBuilder,
     this.backgroundBuilder,
     this.overlayBuilderMap,
+    this.visibleOverlays,
```

Comment:
We have to assert if this prop is specified only if there is a map.

Reply:
I have done it on the initState os the State class, to re use the methods.